lib/utils/manager/storage_directory_manager.dart

```diff
@@ -13,6 +13,8 @@ class StorageDirectoryManager {
 
   static StorageDirectoryManager get instance => _instance;
 
+  factory StorageDirectoryManager() => _instance;
+
   Future<String> getFileStoreDirectory() async {
```

Comment:
IMO, use `StorageDirectoryManager.instance` looks safer, why do have to create a new one

Reply:
please keep only one for easier to use and find